style/react/README.md

```diff
@@ -0,0 +1,50 @@
+# React
+
+ * Use React in [Strict Mode]
+ * Use React with [TypeScript](../typescript/README.md)
+ * Use [Jest](../testing-jest/README.md) for unit testing
+ * Use [styled-components] for CSS-in-JS
+ * Use [Apollo] with [code generation] when working with [GraphQL]
+ * Use [React Router] for routing in SPAs
```

Comment:
This is similar to the css-in-js discussion above. I didn't mean to imply that we _should_ build SPAs, just that if we are the routing solution of choice would be React Router.

But looking at this from the perspective of someone reading these guides for perhaps the first time without that context, it does feel like a recommendation towards SPAs as a default which shouldn't be our stance.

https://github.com/thoughtbot/react-suspenders/pull/1 handles the SPA case right now since it is based off `create-react-app`. Maybe that, too, should have an option for bootstrapping a non-SPA variant, perhaps using rails suspenders. Though I don't have the expertise using those two together 🙂